src/main.rs

```diff
@@ -163,6 +164,7 @@ pub struct Settings {
     spawn_shell: bool,
     tasks: Option<Vec<String>>,
     forced_tasks: Vec<String>,
+    output_path: PathBuf,
```

Comment:
I see it's called "output dir" above but "output path" here. Can we make it consistent? (I prefer dir over path.)